lib/raix/configuration.rb

```diff
@@ -50,6 +50,10 @@ def initialize(fallback: nil)
       self.fallback = fallback
     end
 
+    def client?
```

Comment:
where is this used?

Reply:
The question is how that interact with https://github.com/OlympiaAI/raix?tab=readme-ov-file#global-vs-class-level-configuration, where you can already set a client (or any other config option) either on the global level or the class level. Here the one configured on the class level currently takes precedence. I do wonder if that is the same use case, i.e. different clients for different agent classes or if you want something else?